### ISSUE TITLE
Accessibility issues in the latest version of Google Chrome

### ISSUE THREAD

**Issue by DalisonCPU** [GoogleChrome/lighthouse#11774]
#### Provide the steps to reproduce
1. Run LH on <affected url>
https://www.paypal.com/myaccount/transfer/homepage

#### What is the current behavior?
When I press enter after typing the email of the recipient I want to send the money to, the fields where I need to enter the transfer amount and the note, address are all unavailable (as if they did not exist on the page for the screen reader I am using (nvda access [www.nv-access.org])
#### What is the expected behavior?
after typing the recipient's email and pressing enter or clicking the next button, the fields where I must type the amount I want to send and the note, and also the address selection should be shown with the screen reader I'm using, when use the arrows and the tab.

#### Environment Information
* Affected Channels: 
do not know
* Lighthouse version:
* Chrome version: 
87.0.4280.88 (Versão oficial) 64 bits
* Node.js version:
do not know
* Operating System:
Windows 10
**Related issues**
in version 86 of Google Chrome this did not happen.
I am using the NVDA screen reader which is open source
https://github.com/nvaccess/

**Comment by patrickhulce** [comment]
Thanks for filing @DalisonCPU! I'm not sure I understand how this is related to Lighthouse though. It sounds like you're describing a screen reader or Chromium bug which has nothing to do with Lighthouse results, is that correct?

**Comment by DalisonCPU** [comment]
yea. Do I need to report this problem on chromeun?

**Comment by patrickhulce** [comment]
Ah OK, yes I would start with [nvaccess](https://github.com/nvaccess/) first, but if they conclude that Chromium is the problem, then yes you can file at crbug.com.
